src/main/java/org/mskcc/cmo/metadb/service/impl/ValidRequestCheckerImpl.java

```diff
@@ -101,6 +101,31 @@ public String getFilteredValidRequestJson(String requestJson) throws IOException
         return null;
     }
 
+    /**
+     * Evaluates request metadata and returns a boolean based on whether the request data
+     * passes all sanity checks.
+     * @throws JsonProcessingException or JsonMappingException
+     */
+    @Override
+    public Boolean isValidRequestMetadataJson(String requestJson)
+            throws JsonMappingException, JsonProcessingException {
+        if (StringUtils.isAllBlank(requestJson)) {
+            return false;
+        }
+        ObjectMapper mapper = new ObjectMapper();
+        Map<String, Object> requestJsonMap = mapper.readValue(requestJson, Map.class);
+        boolean isCmoRequest = isCmoRequest(requestJsonMap);
+        boolean hasRequestId = hasRequestId(requestJsonMap);
```

Comment:
[requested change] This is something that can be generalized to handle both the LimsRest schema and our universal schema. Fields that should be checked for the request id: `requestId` and `igoRequestId`